InnerEye/ML/deep_learning_config.py

```diff
@@ -362,6 +362,13 @@ class DeepLearningConfig(GenericConfig, CudaAwareConfig):
                                                  "weights are updated using mean_teacher_"
                                                  "weight = alpha * (mean_teacher_weight) "
                                                  " + (1-alpha) * (current_student_weights). ")
+    weights_url: str = param.String(doc="If provided, a url from which weights will be downloaded and used for model "
+                                        "initialization.")
+    local_weights_path: Optional[Path] = param.ClassSelector(class_=Path,
```

Comment:
to simplify things, why no use the folder "model_weights" you have already defined above?

Reply:
oh, get it now. You are setting that after downloading. Maybe add to doc that this get overwritten when specitfying a URL?

Reply:
I've added a check to make sure both weights_url and local_weights_path cannot be set and clarified this in the doc string, does that fix the confusion around the params?